client/src/components/molecules/RequestTypeDropdown.tsx

```diff
@@ -70,7 +70,7 @@ const RequestTypeDropdown: FunctionComponent<Props> = (props: Props) => {
     };
 
     useEffect(() => {
-        setNumRequests(props.requests!.reduce((total, request) => (request.deleted === false ? total + 1 : total), 0));
+        setNumRequests(props.requests!.reduce((total, request) => (request.deletedAt == null ? total + 1 : total), 0));
```

Comment:
Looks good — just curious what the rationale was for changing this :O

Reply:
To speed it up eventually we only need to pull what is inside embeddings. deletedAt is in embeddings

Reply:
deletedAt and fulfilledAt are in requestEmbeddings in requestTypes and could potentially be pulled without fetching the request documents. I tried to make these changes to speed up the data loading here, but it couldn't be done, and I just left the changes